gems/pending/appliance_console/logical_disk_volume.rb

```diff
@@ -0,0 +1,107 @@
+require 'active_record'
+require 'active_support/core_ext'
+require 'linux_admin'
+require 'pathname'
+require 'util/miq-password'
+require 'fileutils'
+require 'appliance_console/errors'
+require 'appliance_console/utilities'
+require 'appliance_console/logging'
+
+RAILS_ROOT ||= Pathname.new(__dir__).join("../../../")
+
+module ApplianceConsole
+  class LogicalDiskVolume
+    include ApplianceConsole::Logging
+
+    # Required instantiation parameters
+    attr_accessor :disk, :mount_point, :name
+
+    # Derived or optionally provided instantiation parameters
+    attr_accessor :volume_group_name, :filesystem_type, :logical_volume_path
+
+    # Logical Disk creation objects
+    attr_accessor :logical_volume, :partition, :physical_volume, :volume_group
```

Comment:
Are we planning on setting these from outside of the instance or can these be `attr_reader`s (or just regular instance variables)?